packages/form-builder/src/feedback/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React, {useEffect, useState} from "react";
+import {__} from '@wordpress/i18n';
+import Container from './Container'
+import FeedbackButton from "./FeedbackButton";
+
+import {
+    Container as PopupContainer,
+    Header as PopupHeader,
+    Content as PopupContent,
+} from './popup'
+
+const feedbackUrl = 'https://feedback.givewp.com/next-gen';
+
+const HIDE_FEEDBACK = 'givewpNextGenHideFeedback'
+
+const Feedback = () => {
+
+    const [ hidden, setHidden ] = useState(true)
+    const closeCallback = () => {
+        setHidden(true)
+        localStorage.setItem(HIDE_FEEDBACK, 'true')
+    }
+
+    useEffect(() => {
+        setHidden(!! localStorage.getItem(HIDE_FEEDBACK))
+    })
+
+    return (
+        <Container>
+            {!hidden && (
+                <PopupContainer>
+                    <PopupHeader title={__('Have feedback?', 'givewp')} closeCallback={closeCallback} />
+                    <PopupContent>
+                        {__('Let us know what you think about the form builder to help improve the product experience.', 'givewp')}
+                        {' '}
+                        <a href={feedbackUrl} target="_blank" rel="noopener noreferrer" onClick={closeCallback} style={{color: 'var(--give-primary-600)'}}>
+                            {__('Click here', 'givewp')}
```

Comment:
The textdomain is `give` not `givewp`.  We'll probably want to update that eventually but lets just make sure to use the right one for now.

Reply:
Resolved by https://github.com/impress-org/givewp-next-gen/pull/96/commits/0eed4d9f356c9d7ace1a90cca090ba402fbaf163.